doc/source/ray-core/doc_code/limit_tasks.py

```diff
@@ -1,47 +1,38 @@
-import numpy as np
-
+# __without_backpressure_start__
 import ray
 
 ray.init()
 
 
-# __defining_actor_start__
 @ray.remote
 class Actor:
-    def heavy_compute(self, large_array):
+    async def heavy_compute(self):
```

Comment:
Why async? Did the example not work previously?

Reply:
Yea, I chose to use async so tasks are finished in a different order than the submission order so using `ray.wait()` makes more sense otherwise we could just `ray.get()` one by one.